src/Microsoft.ML.Data/Transforms/ColumnSelecting.cs

```diff
@@ -48,7 +55,7 @@ public sealed class ColumnSelectingEstimator : TrivialEstimator<ColumnSelectingT
         /// <param name="env">Instance of the host environment.</param>
         /// <param name="keepColumns">The array of column names to keep.</param>
         private ColumnSelectingEstimator(IHostEnvironment env, params string[] keepColumns)
```

Comment:
>private [](start = 8, length = 7)

I don't see any use of this constructor except for being used at line 90. Since you are cleaning up the code as well, I would suggest removing it.

Reply:
Good point! I removed that

---
In reply to: [254552460](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254552460) [](ancestors = 254552460)